google/cloud/security/common/data_access/sql_queries/snapshot_cycles_sql.py

```diff
@@ -24,6 +24,7 @@
         `complete_time` datetime DEFAULT NULL,
         `status` enum('SUCCESS','RUNNING','FAILURE',
                       'PARTIAL_SUCCESS','TIMEOUT') DEFAULT NULL,
+        `has_all_data` bool DEFAULT NULL,
```

Comment:
Sorry if I didn't explain this clearly in the PR description.  So, the status fields refer to the pipelines themselves, whether they finished running or not (i.e. crashed).  So, it is possible for the pipelines to continue running and considered to be ```successful```, even if there are problems to retrieve subsets of data (e.g. the iam policies of a project) along the way.  That is why we need some way to know if data has been all retrieved or not.  We could change our definition of the current ```status``` to refer to data as well.  Please let me know.